ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -0,0 +1,178 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: operator-release-pipeline
+spec:
+  params:
+    - name: git_repo_url
+    - name: git_revision
+      default: "main"
+    - name: imagestream
+    - name: tag
+    - name: bundle_path
+    - name: dist_method
+    - name: is_latest
+    - name: registry
+      default: quay.io
+    - name: image_namespace
+      default: $(context.pipelineRun.namespace)
+      description: The namespace/organization all built images will be pushed to.
+    - name: pyxis_url
+  workspaces:
+    - name: repository
+    - name: ssh-dir
```

Comment:
Can we make this optional? It's only used to checkout the source which can be done with https.

Reply:
What can be the use case to trigger the internal pipelines different way than intended by design? 
Speaking of which, we would have to include other ways of authentication in docs.
Overall, even ssh-dir might not be necessary here, since we expect to check out only publicly visible repositories...

Reply:
It's very unlikely this will ever be used with private repos but leaving this optional will allow us to support that. Otherwise the repo will always be public with no need for write access.